store.html

```diff
@@ -0,0 +1,119 @@
+        </a>
+        <div class="free-space2"></div>
+    </nav>
+    <main class="form">
+        <section class="form-items">
+            <form class="order-form"  action="">
+                <h1 class="order-karma-title">Order your Karma wifi <br> device today!</h1>
+                <div class="form-line">
+                    <div class="form-items-holder">
+                 <label for="firstname" class="form-input">First name *</label>
+                 <input id="firstname" type="text" name="firstname" required maxlength="15">
+                    </div>
+                    <div class="store-free-space"></div>
+                    <div class="form-items-holder">
+                 <label for="lastname" class="form-input">Last name *</label>
+                 <input id="lastname" type="text" name="lastname" required maxlength="15">
+                    </div>
+                 </div>
+                <div class="form-line">
+                    <div class="form-items-holder">
+                 <label for="address" class="form-input">Address *</label>
+                 <input id="address" type="text" name="address" required class="address">   
+                    </div>
+                </div>
+                <div class="form-line">
+                    <div class="form-items-holder">
+                 <label for="address2" class="form-input">Address 2</label>
+                 <input id="address2" type="text" name="address2" class="address">
+                    </div>
+                </div>
+                <div class="form-line">
+                    <div class="form-items-holder">
+                 <label for="city" class="form-input">City *</label>
+                 <select id="city" name="city" required>
+                    <option value="-1" selected>Select your city...</option>
+                    <option value="london">London</option>
+                    <option value="milan">Milan</option>
+                    <option value="birmingham">Birmingham</option>
+                    <option value="paris">Paris</option>
+                 </select>
+                    </div>
+                    <div class="store-free-space"></div>
+                    <div class="form-items-holder">
+                <label for="postcode" class="form-input">Postcode *</label>
+                <input id="postcode" type="text" name="postcode" maxlength="8" style="text-transform:uppercase">
+                    </div>
+                </div>
+                <div class="form-line">
+                     <fieldset class="form-input" id="select-colour">
+                        <legend>Select a colour</legend>
+                      <div class="radio-btns">
+                          <div class="radio-lebel-holder">
+                            <input id="orange" type="radio" name="orange" value="orange">
+                            <label for="orange" class="radio-label">Karma orange</label>
```

Comment:
I know this is and extra task to change the color of the radio button, but i have a code for you here that might help:
If you give the radio buttons a class and assign this to your css you might get it to work. (it worked for me atleast)

.ENTER-CLASS-HERE 
{
	accent-color: rgb(255, 99, 43);
}